pkg/k8sapi/discovery.go

```diff
@@ -80,8 +81,21 @@ func NewController(clientset kubernetes.Interface) *Controller {
 }
 
 // CreateKubeClient creates a k8s client
-func CreateKubeClient() (clientset.Interface, error) {
-	kubeClient := k8sclient.GetKubeClient()
+func CreateKubeClient(kubeconfig, master string) (clientset.Interface, error) {
```

Comment:
This is called from cni-metrics-helper, and is the reason your unit tests are failing:

```
# github.com/aws/amazon-vpc-cni-k8s/cni-metrics-helper
cni-metrics-helper/cni-metrics-helper.go:82:44: not enough arguments in call to k8sapi.CreateKubeClient
```